### DISCUSSION TITLE
Short GIFs speed up

### DISCUSSION THREAD

**Discussion by Thomas33315** [d2phap/ImageGlass#1345] [General]
I'd like to ask if we can change GIF speed. In Photos it's normal speed while Image Glass speeds it up.

https://user-images.githubusercontent.com/46522120/170414005-454801eb-b2a8-4194-a205-885ea37db6db.mp4

**Comment by d2phap** [comment]
can you send me the GIF file? Thank you!

**Reply by Thomas33315** [reply]
Yes! Here you are
![4](https://user-images.githubusercontent.com/46522120/170814810-cb525a61-dc5e-4a64-a882-f87d8491c22f.gif)

**Comment by d2phap** [comment]
I just checked the GIF, it contains all time frames at 10ms which is very fast.
So IG switches every frame at 10ms which is correct.

I don't know how other apps handle this yet, IrfanView also plays this file as same as IG.
I will investigate more in the future.

![image](https://user-images.githubusercontent.com/3154213/170863192-73ca9056-bfeb-4e07-80a8-7483f43cac44.png)
